newton/examples/example_anymal_c_walk.py

```diff
@@ -17,6 +17,7 @@
 # Example Anymal C walk
 #
 # Shows how to control Anymal C with a pretrained policy.
+# uv run --with torch newton/examples/example_anymal_c_walk.py
```

Comment:
I would add a tiny sentence for context for the user about why they need to use this command